Documentation/Menu.md

```diff
@@ -42,7 +42,7 @@ Some tips will have an offset on their readings, to calibrate this out, perform
 
 1. Connect power to your iron
 2. **Critical: make sure the tip & handle is at room temperature** (wait a good while after using the iron before calibration)
-3. Using (`-/B`) Scroll down to Advanced menu > press (`+/A`) > Calibrate temperature? > press (`+/A`)
+3. Using (`-/B`) scroll down to Advanced settings > press (`+/A`) to select > then (`-/B`) to scroll to Calibrate temperature? > press (`+/A`) until you see ...
```

Comment:
```suggestion
3. Using (`-/B`) scroll down to Advanced settings > press (`+/A`) to select > then (`-/B`) to scroll to Calibrate temperature? > press (`+/A`) until you see **`...`**
```

Reply:
will change thanks this makes that much more visable.